dash_auth/plotly_auth.py

```diff
@@ -150,6 +152,21 @@ def wrap(*args, **kwargs):
     def check_view_access(self, oauth_token):
         return check_view_access(oauth_token, self._dash_app['fid'])
 
+    def get_kerberos_ticket_cache(self):
```

Comment:
This method needs the request context since it uses `flask.request`, there's a decorator I made to assert that.

https://github.com/plotly/dash-auth/blob/70fba0de1b64a115e97cfa03e6466168eb99725f/dash_auth/oauth.py#L14-L22

Reply:
Good idea, thanks for pointing that out.